### ISSUE TITLE
Unable to add a command

### ISSUE THREAD

**Issue by SamLukeYes** [GSConnect/gnome-shell-extension-gsconnect#1095]
**Describe the bug**

When clicking on `+` on the `Commands` page, nothing happens, so remote commands can't be added.

**Steps To Reproduce:**

1. Go to GSConnect Preferences
2. Click on `Commands`
3. Click on `+`
4. Nothing happens

**Expected behavior**

A dialog show up so that I can set a custom command.

**Support Log**

Clicked on `+` for 3 times, and the following error appeared 3 times:
```
4月 10 01:09:52 gjs[635774]: JS ERROR: TypeError: this.get_root is not a function
                              _onEditCommand@/home/yes/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/preferences/device.js:686:37
                              @/home/yes/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/gsconnect-preferences:112:21
```

**System Details (please complete the following information):**

 - **GSConnect version:** 46
   - **Installed from:** GNOME Extensions Website
   - **GNOME/Shell version:** 40.0
 - **Distro/Release:** Arch

**GSConnect environment (if applicable):**

 - **Paired Device(s):** MI MAX 3
 - **KDE Connect app version:** 1.16.0

**Comment by andyholmes** [comment]
Looks like some Gtk4 functions were left after the transition. Should be fixed in #1096, thanks for reporting!

**Comment by SamLukeYes** [comment]
> Should be fixed in #1096, thanks for reporting!

The latest git version can add commands now. Thanks for fixing it! :)
